.github/workflows/tests.yaml

```diff
@@ -31,36 +20,44 @@ jobs:
         done;
 
     - name: load test GET endpoint
-      run: |
-        set -e
-        iter8 launch -c load-test-http --set url=http://127.0.0.1/get
-        cat experiment.yaml
-        iter8 report
-        iter8 assert -c completed -c nofailure -c slos
+      run: | 
+        cat << EOF > experiment-config.yaml
+          url: http://127.0.0.1/get
```

Comment:
Host information should be set in the same manner as gRPC -- extracting the address from docker bridge seems like the more robust approach